cmd/minikube/cmd/start_flags.go

```diff
@@ -133,6 +133,7 @@ const (
 	extraDisks              = "extra-disks"
 	certExpiration          = "cert-expiration"
 	binaryMirror            = "binary-mirror"
+	disableOptimizations    = "disable-optimizations"
```

Comment:
I thinking if this should also be added to the cluster config ? or not hm.... not sure

Reply:
It is currently included in the cluster config, my reasoning is that if they stop and start it again and it's not in the config it's going to start enabling the optimizations again.

Reply:
gotcha, missed that. that sounds good